### PR TITLE
Support for timeseries database / visualization

### PR DESCRIPTION
**Pull request by khusmann**
InfluxDB: https://docs.influxdata.com/influxdb/v1.5/
Chronograf: https://docs.influxdata.com/chronograf/v1.5/
Grafana: https://grafana.com/

You may be less interested in adding Chronograf / Grafana support, let me know and I can remake the pull request with just InfluxDB. (This is my first time trying pull requests; please let me know if this is the proper approach)

Cheers!

### CODE REVIEW

File: scripts/install-chronograf.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+then
+    echo "Chronograf already installed."
+    exit 0
+fi
+
+touch /home/vagrant/.chronograf
+
+wget -q https://dl.influxdata.com/chronograf/releases/chronograf_1.5.0.1_amd64.deb
+sudo dpkg -i chronograf_1.5.0.1_amd64.deb
+
```
Comment (svpernova09) [inline scripts/install-chronograf.sh]: Do we need to `rm chronograf_1.5.0.1_amd64.deb`? We wouldn't want the script to leave the artifact.

**Comment by khusmann** [comment]
You're totally right -- here's a fix!

**Comment by svpernova09** [comment]
Thanks for the PR!

File: scripts/install-chronograf.sh (hunk 1 above)
Comment (Quix0r) [reply scripts/install-chronograf.sh]: And maybe move to it `/tmp/ instead of current directory.

**Pull request merged**